### ISSUE TITLE
We need to add a license!

### ISSUE THREAD

**Issue by hsablonniere** [ensuite/ensuite-present#6]
I would say MIT? The only thing I don't know is what should be put on the first line?

For asciidoctor it's:

```
Copyright (C) 2012-2016 Dan Allen, Ryan Waldron and the Asciidoctor Project
```

@mojavelinux @graphitefriction Should we put my name? OpenDevise? Both?

**Comment by mojavelinux** [comment]
I'm in favor of the MIT license (because I believe open source is first and foremost about freedom).

On the first line, I tend to put the primary author and the community or sponsor. I think we can start with:

```
Copyright (C) 2017 Hubert SABLONNIÈRE and OpenDevise Inc.
```

I like that because it gives ownership to the primary author to make future decisions about the license. Again, that fits my philosophy about open source and gives respect to the creator.

**Comment by hsablonniere** [comment]
@mojavelinux I love it when you don't forget the accent on my lastname!

**Comment by mojavelinux** [comment]
I take equal pride in getting that right.

**Comment by hsablonniere** [comment]
Fixed by #7.
